### PR TITLE
Add home-page to ERP

### PR DESCRIPTION
**Pull request by jainendra**
Home-page is like a virtual desktop
Home-page lists all the apps/modules installed

Solves #16

### CODE REVIEW

**Comment by jainendra** [comment]
@utkarshrpr @nishanthvijayan @ankitdahiya95 Please review it both by the code UI standpoint.

File: home/templates/home/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        </div>
+        <div id="content" class="container">
+            <div id="apps" class="row">
+                <div id="user-management" class="col-md-3 col-sm-4 col-xs-6 white text-center">
+                    <a href="{% url 'home:user-index' %}">
+                        <div class="app-logo center-block circle">
+                            <i class="fa fa-users fa-big box-shadow" aria-hidden="true"></i><br>
+                        </div>
+                        <div class="app-label">
+                            <h4>User-Management</h4>
```
Comment (nishanthvijayan) [inline home/templates/home/index.html]: User Management without the - would look better. I think.

**Comment by nishanthvijayan** [comment]
Could you add a screenshot as well.
Would it look okay if the icons are a bit smaller?

**Comment by nishanthvijayan** [comment]
Everything else looks okay to me!

File: home/templates/home/index.html (hunk 1 above)
Comment (jainendra) [inline home/templates/home/index.html]: Okay. I'll change it

**Comment by nishanthvijayan** [comment]
:+1:

**Pull request merged**